### PR TITLE
Update stellar-sdk to v2.0.1

### PR DESCRIPTION
**Pull request by vcarl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "interstellar-sessions": "~0.0.10",
     "interstellar-ui-messages": "~0.0.3",
     "lodash": "^3.10.1",
-    "stellar-sdk": "^0.14.0"
+    "stellar-sdk": "^2.0.0-rc.2"
```
Comment (bartekn) [inline package.json]: Are you sure there were no breaking changes between these 2 version? Other than that 👍 .

**Comment by morleyzhi** [comment]
The ‘latest’ version should be v2.0.1, not the RC.

**Comment by vcarl** [comment]
Ah okay! I did `yarn add stellar-sdk@next`, `@latest` gets 2.0.1

File: package.json (hunk 1 above)
Comment (vcarl) [reply package.json]: Confirmed that balances + transactions work locally, afaik the breaking changes were in types

**Pull request merged**